utilities/extract_qm_mm_snapshots.py

```diff
@@ -176,6 +176,11 @@ def process_snapshots(args):
         with open(tc_input_filename, "w") as params_file:
             params_file.write(terachem_params(mm_filename, qm_filename))
 
+        gaussian_input_filename = os.path.join(
+            frame_dir,
+            f"gaussian_vee_{args.qm_radius}ang.com",
+        )
+
         dye_coords_list = []
         dye_atom_labels_list = []
```

Comment:
Deep copy the atom labels list to use in the diabatization Gaussian file generation

Reply:
I will add it soon. The list contains atom labels that are alphanumeric, so make sure to remove the integers before saving the .com files.